File: examples/spring-starter/src/test/java/org/camunda/bpm/extension/process_test_coverage/examples/NonCamundaTest.java
package org.camunda.bpm.extension.process_test_coverage.examples;

import org.camunda.bpm.extension.process_test_coverage.engine.ExcludeFromProcessCoverage;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
@ExcludeFromProcessCoverage
public class NonCamundaTest {

    @Test
    public void testSomethingElseAgain() {
        Assertions.assertTrue(true);
    }


}

File: regression-tests/spring-starter-platform-7/src/test/kotlin/org/cibseven/community/process_test_coverage/spring_starter/platform7/NonCamundaTest.kt
package org.cibseven.community.process_test_coverage.spring_starter.platform7

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import


@Configuration
class NonCamundaApplication

@SpringBootTest(classes = [NonCamundaApplication::class])
@Import(CoverageTestConfiguration::class)
class NonCamundaTest {

    @Test
    fun testSomethingNotCamunda() {
        val testString = "Testing"
        Assertions.assertThat(testString).isNotEmpty()
    }

}